src/components/utils/include/utils/file_system.h

```diff
@@ -43,81 +42,78 @@ namespace file_system {
 
 /**
  * @brief Get available disc space.
- *
- * @param path to directory
+ * @param utf8_path path to directory
  * @return free disc space.
  */
-uint64_t GetAvailableDiskSpace(const std::string& path);
+uint64_t GetAvailableDiskSpace(const std::string& utf8_path);
 
 /*
  * @brief Get size of current directory
- *
- * @param path to directory
+ * @param utf8_path path to directory
  */
-size_t DirectorySize(const std::string& path);
+size_t DirectorySize(const std::string& utf8_path);
 
 /*
  * @brief Get size of current file
- *
- * @param path to file
+ * @param utf8_path path to file
  * @return size of file, return 0 if file not exist
  */
-int64_t FileSize(const std::string& path);
+int64_t FileSize(const std::string& utf8_path);
 
 /**
  * @brief Creates directory
- * @param name path to directory
+ * @param utf8_path path to directory
  * @return path to created directory.
  */
-std::string CreateDirectory(const std::string& name);
+std::string CreateDirectory(const std::string& utf8_path);
 
 /**
  * @brief Creates directory recursively
- * @param path - full path to directory
+ * @param utf8_path full path to directory
  * @return return true if directory was created or already exist
  */
-bool CreateDirectoryRecursively(const std::string& path);
+bool CreateDirectoryRecursively(const std::string& utf8_path);
 
 /**
   * @brief Checks the file to see whether the file is a directory
-  * @param name path to file
+  * @param utf8_path path to file
   * @return returns true if file is directory.
   */
-bool IsDirectory(const std::string& name);
+bool IsDirectory(const std::string& utf8_path);
 
 /**
   * @brief Is directory exist
-  * @param name path to directory
+  * @param utf8_path path to directory
   * @return returns true if directory is exists.
   */
-bool DirectoryExists(const std::string& name);
+bool DirectoryExists(const std::string& utf8_path);
 
 /**
   * @brief Is file exist
-  * @param name path to file
+  * @param utf8_path path to file
   * @return returns true if file is exists.
   */
-bool FileExists(const std::string& name);
+bool FileExists(const std::string& utf8_path);
 
 /**
   * @brief Writes to file
   *
   * @remark - create file if it doesn't exist
-  * @param name path to file
+  * @param utf8_path path to file
   * @param data data to write
   * @return returns true if the operation is successfully.
   */
-bool Write(const std::string& file_name,
+bool Write(const std::string& utf8_path,
```

Comment:
Why we need to rename the whole parameters to UTF8?
I mean if it will break the ASCII support - yes, but we just extend it. Comments seems to be enough.